graylog2-web-interface/src/pages/IndexSetPage.jsx

```diff
@@ -132,12 +132,16 @@ const IndexSetPage = React.createClass({
     if (this.state.indexerOverview && this.state.indexDetails.closedIndices) {
       const deflectorInfo = this.state.indexerOverview.deflector;
 
+      let activeIndex = ".";
```

Comment:
Please use single quotes for strings (our eslint config should flag this as an error).